sdk/src/main/resources/account/api/accountApi.yaml

```diff
@@ -1389,6 +1389,320 @@ paths:
+            schema:
+              type: object
+              required:
+                - blockSignPubKey, vrfPubKey ,rewardShare, rewarAddress
+              properties:
+                blockSignPubKey:
+                  type: string
+                  description: Key of this forger (private key must be in the local wallet)
+                vrfPubKey:
+                  type: string
+                  description: Vrfkey of this forger(private key must be in the local wallet)
+                rewardShare:
+                  type: integer
+                  description: Reward to be redirected to a separate reward address (integer, range from 0 to 1000 - where 1000 represents 100%)
+                rewarAddress:
+                  type: string
+                  description: External reward address (may be a single EOA or (more likely) a smart contract handling rewards distribution to delegators) |
+                gasInfo:
+                  $ref: '#/components/schemas/EIP1559GasInfo'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  result:
+                    type: object
+                    properties:
+                      transactionId:
+                        type: string
+                  error:
+                    $ref: '#/components/schemas/SidechainApiErrorResponse'
+        default:
+          description: any kind of http error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SidechainApiError'
+                
+  /transaction/pagedForgingStakes:
+    post:
+      tags:
+        - transaction
+      summary:  Returns the paginated list of forging stakes.
+      description: Returns the paginated list of forging stakes.
+      operationId: pagedForgingStakes
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - blockSignPubKey, vrfPubKey ,rewardShare, rewarAddress
+              properties:
+                startPos:
+                  type: integer
+                  description: Start position in the paginated list
+                  default: 0
+                size:
+                  type: integer
+                  description: Number of records to return
+                  default: 10
+                gasInfo:
+                  $ref: '#/components/schemas/EIP1559GasInfo'
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  result:
+                    type: object
+                    properties:
+                      nextPos:
+                        type: integer
```

Comment:
Is it possible to specify what nextPos means? Eg that if it is == -1 means no additional records, otherwise it is the starting index of the remaining elements.
Same for other paged methods